.github/ISSUE_TEMPLATE.md

```diff
@@ -24,4 +24,8 @@ Describe what you expect the output to be. Knowing the correct behavior is also
 Provide any additional information here.
 
 #### Current Version:
+<<<<<<< HEAD
```

Comment:
A lot of conflicts. I think the latest merge into these release branches needs to be un-done